lib/child_process.js

```diff
@@ -331,7 +320,23 @@ function normalizeSpawnArguments(file /*, args, options*/) {
   else if (options === null || typeof options !== 'object')
     throw new TypeError('"options" argument must be an object');
 
-  options = util._extend({}, options);
+  // Make a shallow copy so we don't clobber the user's options object.
+  options = Object.assign({}, options);
+
+  if (options.shell) {
+    const command = [file].concat(args).join(' ');
```

Comment:
Ah, looks like the `/s` enables that behavior - http://stackoverflow.com/questions/9866962/what-is-cmd-s-for